js/index.js

```diff
@@ -0,0 +1,28 @@
+// !! IMPORTANT README:
```

Comment:
It is a good practice to have a gitignore file to not import files such as tests. That way it takes less storage.